### DISCUSSION TITLE
Can I disable HTTPS when the docker nginx hosts the website?

### DISCUSSION THREAD

**Discussion by skeet16** [dfir-iris/iris-web#593] [Q&A] [answered]
Can I host IRIS as a http website? Please help with this.

**Comment by whikernel** [comment]
Hey! 
I'm not sure I understand the question. Could you please be a bit more specific?

**Reply by skeet16** [reply, marked as answer]
Hey, sorry for the misunderstanding. I figured it out. I wanted to use my own certificates for hosting IRIS. I found the CERT_FILENAME and KEY_FILENAME later on which allowed me to use my own certs. Thanks for the reply tho.
